ui/src/components/Drawer/components/FileManagement/index.vue

```diff
@@ -230,6 +230,7 @@ const createColumns = (): DataTableColumns<RowData> => {
       title: t('Type'),
       key: 'type',
       align: 'center',
+      width: 100,
       render(row: RowData) {
         return h(
           NText,
```

Comment:
There are no known differences or irregularities in the provided code snippet related to potential issues, optimizations, etc. The code appears to be syntactically correct and follows modern coding best practices. However, please ensure that the current implementation aligns with the latest technology stack and requirements of your project before deploying it. Always test extensively during development stages for quality assurance.